src/Symfony/Component/Form/Tests/Extension/Core/Type/ChoiceTypeTest.php

```diff
@@ -1272,38 +1139,12 @@ public function testSubmitMultipleExpandedObjectChoices()
         $this->assertNull($form[4]->getViewData());
     }
 
-    public function testSubmitMultipleExpandedNumericChoices()
```

Comment:
This test should trigger a deprecatio warning shouldn't it in 2.8? But it's not marked as legacy. And there does not seem to be an equivalent test for the new way.